webapp/channels/src/packages/mattermost-redux/src/utils/user_utils.ts

```diff
@@ -186,6 +186,23 @@ export function filterProfilesStartingWithTerm(users: UserProfile[], term: strin
     });
 }
 
+export function filterProfilesStartingWithTermOrPositionMatchingWithTerm(users: UserProfile[], term: string): UserProfile[] {
```

Comment:
While the self-explanatory name is good, I feel like it might be better to just go with something simpler and let the exact filtering be an implementation detail. That said, that name would depend on whether or not we want to change this filtering elsewhere